Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -5,2169 +5,2819 @@ name: Recorded Future v2
-  name: file_threshold
-  defaultvalue: "65"
-  type: 0
-  required: false
-- display: CVE Threshold. Minimum risk score from Recorded Future to consider the
-    CVE malicious.
-  name: cve_threshold
-  defaultvalue: "65"
-  type: 0
-  required: false
-- display: IP Threshold. Minimum risk score from RF to consider the IP malicious.
-  name: ip_threshold
-  defaultvalue: "65"
-  type: 0
-  required: false
-- display: Domain Threshold. Minimum risk score from Recorded Future to consider the
-    domain malicious.
-  name: domain_threshold
-  defaultvalue: "65"
-  type: 0
-  required: false
-- display: URL Threshold. Minimum risk score from Recorded Future to consider the
-    URL malicious.
-  name: url_threshold
-  defaultvalue: "65"
-  type: 0
-  required: false
-- display: Vulnerability Threshold. Minimum risk score from Recorded Future to consider
-    the vulnerability critical.
-  name: vulnerability_threshold
-  defaultvalue: "65"
-  type: 0
-  required: false
-- display: 'Trust any certificate (not secure)'
-  name: unsecure
-  defaultvalue: "false"
-  type: 8
-  required: false
-- display: Use system proxy settings
-  name: proxy
-  defaultvalue: "false"
-  type: 8
-  required: false
-- display: Fetch incidents
-  name: isFetch
-  required: false
-  type: 8
-- display: Rule names to fetch alerts by, separated by semicolon. If empty, all alerts
-    will be fetched
-  name: rule_names
-  required: false
-  type: 12
-- defaultvalue: 24 hours
-  display: First fetch time (<number> <time unit>, e.g., 12 hours, 7 days, 3 months,
-    1 year)
-  name: first_fetch
-  required: false
-  type: 0
-- display: Incident type
-  name: incidentType
-  required: false
-  type: 13
-- display: Maximum number of incidents per fetch
-  name: max_fetch
-  defaultvalue: "50"
-  required: false
-  type: 0
-- additionalinfo: Reliability of the source providing the intelligence data.
-  defaultvalue: B - Usually reliable
-  display: Source Reliability
-  name: integrationReliability
-  options:
-  - A+ - 3rd party enrichment
-  - A - Completely reliable
-  - B - Usually reliable
-  - C - Fairly reliable
-  - D - Not usually reliable
-  - E - Unreliable
-  - F - Reliability cannot be judged
-  required: false
-  type: 15
-- defaultvalue: indicatorType
-  name: feedExpirationPolicy
-  display: ''
-  options:
-  - never
-  - interval
-  - indicatorType
-  - suddenDeath
-  required: false
-  type: 17
-- defaultvalue: '20160'
-  name: feedExpirationInterval
-  display: ''
-  required: false
-  type: 1
+  - display: Server URL (e.g., https://api.recordedfuture.com/gw/xsoar/)
+    name: server_url
+    defaultvalue: https://api.recordedfuture.com/gw/xsoar/
+    type: 0
+    required: true
+  - display: API Token
+    name: token
+    defaultvalue: ""
+    type: 4
+    required: true
+  - display: File Threshold. Minimum risk score from Recorded Future to consider the file malicious.
+    name: file_threshold
+    defaultvalue: "65"
+    type: 0
+    required: false
```

Comment:
```suggestion
  - display: File Threshold.
    additionalinfo: Minimum risk score from Recorded Future to consider the file malicious.
    name: file_threshold
    defaultvalue: "65"
    type: 0
    required: false
```

The display of a param should be short and include only the name of the param.
If you wish to add any information about the param such as what is it used for / what kind of input it's expected to receive you should add it to the 'additionalinfo' key.

Please fix this for the rest of the integration parameters and function arguments.
Also, please fix the RN to include the new display names and also re-generate the readme file so it'll include the new fixes.

Reply:
Done, but Pull request is not fetching my base branch changes, for some reason